blog/2025-06-17-deploying-tolgee-on-sealos.mdx

```diff
@@ -0,0 +1,72 @@
+---
+slug: deploying-tolgee-on-sealos
+title: 'Deploying Tolgee on Sealos'
+description: 'You can now self-host Tolgee on Sealos with just one click'
+image: '/img/blog/deploying-tolgee-on-sealos/sealos-hosting-thumbnail-dark.webp'
+authors: [jcizmar]
+tags: ['tolgee', 'sealos', 'self-host', 'localization', 'guide']
+---
+
+import { ScreenshotWrapper } from '../platform/shared/_ScreenshotWrapper';
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<ThemedImage
+  alt="Apple String Catalog Support Blog Image"
+  sources={{
+    light: useBaseUrl(
+      '/img/blog/deploying-tolgee-on-sealos/sealos-hosting-thumbnail-light.webp'
+    ),
+    dark: useBaseUrl(
+      '/img/blog/deploying-tolgee-on-sealos/sealos-hosting-thumbnail-dark.webp'
+    ),
+  }}
+/>
+
+This guide shows how to self-host Tolgee in two ways: using [Sealos](https://sealos.io/), or deploying with ways that give you more control over your instance. This will help you determine the best option for your needs.
+
+<!--truncate-->
+
+Localization is key for reaching international audiences. Tolgee is an open-source localization platform that offers powerful features for translating applications with support for React, Angular, Vue, and even has a [Figma plugin](https://tolgee.io/apps-integrations/figma-plugin). Tolgee provides an excellent cloud-hosted solution, but some organizations prefer self-hosting for enhanced control and customization.
+
+## Why Self-Host?
+
+Self-hosting Tolgee provides organizations with complete control, the freedom to modify and customize the platform according to specific needs, and integration within existing infrastructure. However, it comes with its drawbacks, as you know, “with great power comes great responsibility,” which means that you will be dealing with all the setup and problems that occur with modifications that you are doing with the self-hosted version. Unlike on Tolgee cloud, you get limited support, and you need to use more intricate ways to work with it. You can upgrade to a more advanced plan for self-hosted, which also provides webhooks, standard support, custom content storage, and AI customization
```

Comment:
Missing "." at the end.
PS: When I run it trhough Grammarly, I see a few issues with commas etc. Could you check it with Grammarly?